a3fe/tests/README.md

```diff
@@ -6,23 +6,26 @@ This directory contains integration tests for A3FE, which are designed to run in
 
 To run integration tests, use the following command:
 
-```bash
-cd your/path/a3fe
-RUN_SLURM_TESTS=1 pytest a3fe/tests --run-integration -v
-```
+   ```bash
+   cd your/path/a3fe
+   RUN_SLURM_TESTS=1 pytest a3fe/tests --run-integration -v
```

Comment:
Maybe you could also mention they can now run "make test-integration" - thanks for adding that!